apps/condo/domains/meter/components/TabContent/Meter.tsx

```diff
@@ -46,6 +48,19 @@ const FULL_WIDTH_DATE_RANGE_STYLE: CSSProperties = { width: '100%' }
 const SORTABLE_PROPERTIES = ['verificationDate', 'source']
 const DEFAULT_DATE_RANGE: [Dayjs, Dayjs] = [dayjs(), dayjs().add(2, 'month')]
 
+
+const getInitialArchivedOrActiveMeter = (router: NextRouter, field: 'isShowActiveMeters' | 'isShowArchivedMeters', defaultValue = false) => {
+    if (field in router.query && isString(router.query[field])) {
```

Comment:
I don't think this is really necessary)
```suggestion
    if (field in router.query && typeof router.query[field] === 'string') {
```